### PR TITLE
Add new parameters to push methods

### PR DESCRIPTION
**Pull request by jdrichards50**
Add new parameters to existing pull and push data methods and minor cleanup. 

How to test. (you must be using the stx/35462_mobile_push_methods branch)

1. Comment out quotefeed in sdk template.
2. Put the below code in ChartViewController.chartIQViewDidFinishLoading.
3. Put a pushUpdate call right after, change the data.

```
var ohlc = ChartIQData(date: Date(), open: 45.5, high: 45.5, low: 45.5, close: 45.5, volume: 50000, adjClose: 45.5)
var data: [ChartIQData] = [ohlc]
chartIQView.push("IBM", data: data)
chartIQLoadedInitialData()
```

### CODE REVIEW

File: Source/ChartIQScriptManager.swift [hunk 1]
```diff
@@ -418,8 +419,8 @@ internal class ChartIQScriptManager: ChartIQScriptManagerProtocol {
   /// - Parameters:
   ///   - jsonString: The String Object.
```
Comment (mwilber-ciq) [inline Source/ChartIQScriptManager.swift:420]: Update parameter description

File: Source/ChartIQScriptManager.swift [hunk 2]
```diff
@@ -653,8 +654,8 @@ internal class ChartIQScriptManager: ChartIQScriptManagerProtocol {
   ///   - moreAvailable: The Bool Value.
   ///   - cb: The String Object.
```
Comment (mwilber-ciq) [inline Source/ChartIQScriptManager.swift:655]: Update parameter description

File: Source/ChartIQView.swift [hunk 3]
```diff
@@ -526,27 +526,27 @@ public class ChartIQView: UIView {
   ///
   /// - Parameters:
   ///   - data: An array of properly formatted OHLC quote objects to create a chart.
```
Comment (mwilber-ciq) [inline Source/ChartIQView.swift:528]: Same for these docs

File: Source/ChartIQScriptManager.swift (hunk 1 above)
Comment (jdrichards50) [reply Source/ChartIQScriptManager.swift:420]: whoops, totally forgot about these.

File: Source/ChartIQScriptManager.swift [hunk 4]
```diff
@@ -650,9 +652,10 @@ internal class ChartIQScriptManager: ChartIQScriptManagerProtocol {
   /// Returns a script that formats JSQuote data.
   ///
   /// - Parameters:
-  ///   - json: The String Object.
-  ///   - moreAvailable: The Bool Value.
-  ///   - cb: The String Object.
+  ///   - data: data: The data you want to display on the chart as a JSON string.
```
Comment (mwilber-ciq) [inline Source/ChartIQScriptManager.swift:655]: Remove the extra `data:`

**Review by mwilber-ciq — CHANGES_REQUESTED** [review]
Just the one edit below. GitHub doesn't seem to do suggested changes with swift.

**Review by mwilber-ciq — APPROVED** [review]

**Pull request merged**